contracts/src/v0.8/dev/automation/2_1/interfaces/IKeeperRegistryMaster.sol

```diff
@@ -89,10 +89,10 @@ interface IKeeperRegistryMaster {
   event UpkeepPerformed(
     uint256 indexed id,
     bool indexed success,
-    uint32 checkBlockNumber,
+    uint96 totalPayment,
     uint256 gasUsed,
     uint256 gasOverhead,
-    uint96 totalPayment
+    bytes trigger
```

Comment:
We are passing here some id based on the trigger right? 
`trigger` might be too generic name

Reply:
the trigger should be one of...
```
```